parm/README.make_flatfile

```diff
@@ -0,0 +1,20 @@
+To make a .txt flat file for UPP, follow these steps within the parm/ directory:
+
+1) Select, then validate, the associated "post_avblflds" file, e.g.,
+
+      xmllint --noout --schema EMC_POST_Avblflds_Schema.xsd fv3lam_post_avblflds.xml
+
+2) Select, then validate, the associated "postcntrl" file, e.g.,
+
+      xmllint --noout --schema EMC_POST_CTRL_Schema.xsd fv3lam_rrfs.xml
+
+3) If file validation succeeds in steps (1) and (2), generate the .txt flat file, e.g.,
+
+      perl PostXMLPreprocessor.pl fv3lam_rrfs.xml fv3lam_post_avblflds.xml postxconfig-NT-fv3lam_rrfs.txt
+
+   In this example, "fv3lam_rrfs.xml" and "fv3lam_post_avblflds.xml" are input files, read by the perl 
+   script. The "postxconfig-NT-fv3lam_rrfs.txt" flat file is the output file produced by the script, which 
+   can have any name.
+
+   Note that a log file will be created by this script, in addition to the flat file.
+
```

Comment:
@EricJames-NOAA There is  a script parm/makefile for generating user defined flat control file with user defined xml and avbflds xml.  The user may execute "make" to get a new flat file when the related xml file has updates. Would you consider to add the rrfs control files in it?

Reply:
@WenMeng-NOAA thanks for pointing it out, I did not know about that.  I will add RRFS control files.